feature/quality-normal: add quality definitions for default intent

ronoaldo opened the pull request:
Uses the Santy_V2.curaprofile which yelds better results than
standard Cura Fine - Normal profiles.

ronoaldo committed:
feature/quality-normal: add quality definitions for normal.

Uses the Santy_V2.curaprofile which yelds better results than
standard Cura Fine - Normal profiles.

ronoaldo committed:
Added basic quality profiles for each resolution.

Matches the basic changes from Flashprint, need real print
tests in order to understand how they work.

ronoaldo commented:
Was able to view the settings this way after loading the files into the quality folder, and after changing the def.json to use machine profiles:

![Captura de tela de 2020-06-01 23-40-18](https://user-images.githubusercontent.com/564649/83474228-df199480-a461-11ea-9e1c-ce7da8c5cb86.png)

ronoaldo commented:
Before merging, kindly let me print tests with these so we make sure I had no changes that could be bad.

ronoaldo committed:
Calibration test optimization.

After printing some calibration tests, I noticed i messed up
with some settings from the original profile, and reverted these
changes.

Small updates:
* removed temperature settings (almost no strings)
* changed 109% to 110% extrusion multiplier, gave best results
  for my prints.

ronoaldo commented:
Print results were amazing but in this test cube I still got some small oozle and strings. I honestly don't know if this is a torture test or the printer would be actually able to print that way:

About the temperature I am open with any suggestions really. It did worked best for the PLA I have here with 195-200º but it could be just me and this particular PLA brand.

Here are some tests with a multi-test calibration cube:

0.18mm https://youtu.be/XE4TnujQkg0
0.08mm https://youtu.be/SjTBDqtHgZ0

eskeyaar commented on quality/finder/finder_hyper.inst.cfg:
```diff
@@ -0,0 +1,57 @@
+
+[values]
+layer_height = 0.08
+layer_height_0 = 0.20
```
i would print my first layer height at  0.1mm else i see there will be elephant foot.

ronoaldo replied:
Makes sense, I'll update the pull request with that.

eskeyaar commented on quality/finder/finder_hyper.inst.cfg:
```diff
@@ -0,0 +1,57 @@
+retraction_combing_max_distance = 2
+retraction_speed = 30
+skin_preshrink = 0.1
+speed_print = 80
```
For 80 Microns , we dont want to print at 80mm/s maybe around 20-25mm/s that will give best result. Do you have .curaprofile of these profiles? if so i can quickly change values and send you back.

ronoaldo replied:
Oh, sorry, i just copied the file to show other resolutions. You are correct this will not look good.

eskeyaar commented on quality/finder/finder_standard.inst.cfg:
```diff
@@ -11,45 +11,47 @@ weight = 0
 skin_preshrink = 0.1
-speed_print = 50
-speed_travel = 100
+speed_print = 80
```
is print coming out good at this speed?

eskeyaar commented on quality/finder/finder_standard.inst.cfg:
```diff
@@ -11,45 +11,47 @@ weight = 0
 support_line_distance = 4.5
-support_z_distance = 0.18
-top_layers = 7
+support_z_distance = =layer_height 
```
nice, but did you print support test prints and checked it?

ronoaldo replied:
Good catch. Do you know of a good test file for testing supports? I'll print a "cube in the air" so that it generates the supports as a first try.

eskeyaar reviewed (commented):
> 
> 
> Print results were amazing but in this test cube I still got some small oozle and strings. I honestly don't know if this is a torture test or the printer would be actually able to print that way:
> 
> About the temperature I am open with any suggestions really. It did worked best for the PLA I have here with 195-200º but it could be just me and this particular PLA brand.
> 
> Here are some tests with a multi-test calibration cube:
> 
> 0.18mm https://youtu.be/XE4TnujQkg0
> 0.08mm https://youtu.be/SjTBDqtHgZ0

Looks good!. i see some vertical Z banding, is your leadscrew lubed properly? 
For the temp, its upto user , they might have different brand which needs different temp, so we leave that alone as of now. 
For the stringging and oozing, crank up the travel speed to more and try and see.

ronoaldo commented:
> Looks good!. i see some vertical Z banding, is your leadscrew lubed properly?

That is a good question! I'll lub it again. This is related to the Z axis correct?

> For the temp, its upto user , they might have different brand which needs different temp, so we leave that alone as of now.

Makes sense.

> For the stringging and oozing, crank up the travel speed to more and try and see.

Great. I'll run some string tests adjusting those values to find the proper one.

ronoaldo committed:
Fixes after review for settings and speed.

Settings for each profile were normalized so they can be
more easily changed.

ronoaldo committed:
Small tweak for finder.def.json

Fallback to .gcode if .gx is not available, since it will be a plugin.

Added small code format to the multiline attributes, just syntax suggar.

ronoaldo committed:
Fixes for the printer def.json mime-type list.

ronoaldo commented:
@eskeyaar I did some string testing yesterday, and it is ok for > 0.18mm, not so well for .12mm and .08mm. Probably this is my fault because I just repeated the values for the configs from the .18 starter one.
